custom_components/tuya_local/devices/beca_zht002_thermostat_c.yaml

```diff
@@ -0,0 +1,93 @@
+name: Zigbee Thermostat
+products:
+  - id: bf63833d92f517329fqkhz
```

Comment:
Is this the product id? Zigbee devices usually have a shorter product id.

Reply:
Got it, since the `product_id` was so short I was not sure